app/src/pages/inside/projectSettingsPageContainer/content/notifications/ruleGroup/ruleGroup.jsx

```diff
@@ -95,6 +95,11 @@ export const RuleGroup = ({ pluginName, typedRules, notifications, isPluginEnabl
 
   const isUpdateSettingAvailable = canUpdateSettings(userRole, projectRole);
   const isReadOnly = !isUpdateSettingAvailable || !isPluginEnabled;
+  const isActivationRequired =
```

Comment:
can be simplified `const isActivationRequired = isUpdateSettingAvailable || typedRules?.length > 0;`